kernel/data-pipeline/scenario/consistencycheck/src/main/java/org/apache/shardingsphere/data/pipeline/scenario/consistencycheck/api/impl/ConsistencyCheckJobAPI.java

```diff
@@ -213,56 +220,85 @@ public void dropByParentJobId(final String parentJobId) {
     }
     
     /**
-     * Get consistency job item info.
+     * Get consistency job item infos.
      *
      * @param parentJobId parent job id
-     * @return consistency job item info
+     * @return consistency job item infos
      */
-    public ConsistencyCheckJobItemInfo getJobItemInfo(final String parentJobId) {
+    public List<ConsistencyCheckJobItemInfo> getJobItemInfos(final String parentJobId) {
```

Comment:
It's better to extract or keep `getJobItemInfo` method (private), and then add `getJobItemInfos` method to reference it